tests/test_level/test_level_04.py

```diff
@@ -572,6 +572,20 @@ def test_print_list_access_index_var(self):
             extra_check_function=check_in_list
         )
 
+    def test_list_access_misspelled_at_gives_error(self):
+        code = textwrap.dedent("""\
+        dieren is Hond, Kat, Kangoeroe
+        print dieren at random""")
+
+        self.multi_level_tester(
+            max_level=5,
+            # set to false for now, probably is incorrect so a look would be appreciated!
```

Comment:
This is correct actually, it disables a special setting we have for skipping error lines. The only thing is that I prefer False for false over 0.